packages/common/src/auditlogs/auditlogslist.js

```diff
@@ -86,15 +76,9 @@ export const AuditLogsList = ({
         </div>
         <div className="auditlogs-list">
           {items.map(item => {
-            const allowsExpansion = !!item.change || item.action.includes('terminal') || item.action.includes('portforward');
             return (
-              <div
-                className={`auditlogs-list-item ${allowsExpansion ? 'clickable' : ''}`}
-                key={`event-${item.time}`}
-                onClick={() => onIssueSelection(allowsExpansion ? item : undefined)}
-              >
+              <div className="auditlogs-list-item" key={`event-${item.time}`}>
                 {auditLogColumns.map((column, index) => column.render(item, index, userCapabilities))}
```

Comment:
This is not the same - as the selection was supposed to happen on a click on the list item. Before Alvaldi the _View details_ was even removed as users would click on the row and to give more space to the content (I think)...
An alternative might be to make the list item click dependent on the `onIssueSelection` being set, so something like:
```jsx
const allowsExpansion = onIssueSelection && (!!item.change || item.action.includes('terminal') || item.action.includes('portforward'));
            return (
              <div
                className={`auditlogs-list-item ${allowsExpansion ? 'clickable' : ''}`}
                key={`event-${item.time}`}
                onClick={() => onIssueSelection ? onIssueSelection(allowsExpansion ? item : undefined) : null}
              >
```

Reply:
I see. Thanks for the feedback. I will adjust it soon